docs/_plugins/shortcodes/example.cjs

```diff
@@ -18,34 +18,41 @@ const path = require('path');
  * @param {string} urlPath
  */
 async function getImg(url, width1x, width2x, outputDir, urlPath) {
-  try {
-    return await Image(url, {
-      urlPath,
-      outputDir,
-      formats: ['auto'],
-      widths: [width1x, width2x],
-      filenameFormat(id, src, width, format) {
-        const extension = path.extname(src);
-        const name = path.basename(src, extension);
-        // rewrite the default 2X image since we don't need two copies
-        return width === width2x ? `${name}.${format}` : `${name}-${width}w.${format}`;
-      },
-    });
-  } catch (error) {
-    return false;
```

Comment:
@nikkimk NB: we should not have swallowed these errors, see next commit for the places which crashing here helped to fix